src/ray/gcs/BUILD.bazel

```diff
@@ -0,0 +1,58 @@
+load("//bazel:ray.bzl", "ray_cc_library")
+
+ray_cc_library(
+    name = "gcs_redis_client",
+    srcs = [
+        "redis_async_context.cc",
+        "redis_client.cc",
+        "redis_context.cc",
+    ],
+    hdrs = [
+        "redis_async_context.h",
+        "redis_client.h",
+        "redis_context.h",
+    ],
+    deps = [
+        "//:hiredis",
+        "//src/ray/common:asio",
+        "//src/ray/common:ray_config",
+        "//src/ray/common:status",
+        "//src/ray/stats:stats_lib",
+        "//src/ray/util:exponential_backoff",
+        "//src/ray/util:util",
```

Comment:
It's the same as `//src/ray/util:util`, actually if you use bazel linter it automatically fixes